### PR TITLE
chore(release): fix rollback command

### PR DESCRIPTION
**Pull request by bobylito**
Incorrect git command for resetting master during a rollback.

### CODE REVIEW

File: scripts/release.js [hunk 1]
```diff
@@ -79,7 +79,7 @@ const strategy = currentBranch !== 'develop' || forceBeta ? 'beta' : 'stable';
 function rollback(newVersion) {
   if (strategy === 'stable') {
     // reset master
-    shell.exec('git reset --hard origin master');
+    shell.exec('git reset --hard origin/master');
```
Comment (Haroenv) [inline scripts/release.js]: what about doing `--soft` instead? this command is indeed the fix though

Comment (bobylito) [reply scripts/release.js]: It will leave the working files there, and I fail to see any value in that. Can you give your opinion?

Comment (Haroenv) [reply scripts/release.js]: not sure either, just wondering if there could be any caution (if you e.g. released before pushing)?

Comment (bobylito) [reply scripts/release.js]: There is no rollback possible if the package is published. As far as I can see this is what we expect.

**Pull request merged**